src/utils.ts

```diff
@@ -234,7 +235,7 @@ export class Utils {
         core.debug('Exchanging GitHub JSON web token with a JFrog access token...');
 
         let projectKey: string = process.env.JF_PROJECT || '';
-        let jobId: string = process.env.GITHUB_JOB || '';
+        let jobId: string = Utils.GitHubJobIdEncoded();
```

Comment:
Please follow the convention of lowercase capital letter for methods. Preferably also use `this` over the class name.
In addition please run `npm run format`